### PR TITLE
Display hfc in "pennies"

### PR DESCRIPTION
**Pull request by davidkelly**
We have decided to display hfc without a decimal for now.  While at it, I now create the transaction history text in the client, converting times to the system time.  Don't get me started about qt's toLocalTime conversion...

### CODE REVIEW

File: interface/src/commerce/Ledger.cpp [hunk 1]
```diff
@@ -108,6 +108,61 @@ void Ledger::inventory(const QStringList& keys) {
...
 }
 
+QString nameFromKey(const QString& key, const QStringList& publicKeys) {
+    if (key.isNull() || key.isEmpty()) {
+        return "<b>Marketplace</b>";
+    }
+    if (publicKeys.contains(key)) {
+        return "You";
+    }
+    return "<b>Someone</b>";
```
Comment (zfox23) [inline interface/src/commerce/Ledger.cpp]: Very nice...

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2975/HighFidelity-Beta-PR11289-396f1081824ff4bc2b5bab40c9a23e9109ba6a22.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2981/HighFidelity-Beta-PR11289-cdadaa8c9c220b15d868095d8e68b43432a2c0fa.exe

File: scripts/system/html/js/marketplacesInject.js [hunk 2]
```diff
@@ -175,11 +175,12 @@
...
             var href = $('#side-info').find('.btn').first().attr('href');
             $('#side-info').find('.btn').first().attr('href', '#');
-            
+
             var cost = $('.item-cost').text();
 
             if (parseInt(cost) > 0 && $('#side-info').find('#buyItemButton').size() === 0) {
-                $('#side-info').find('.btn').first().html('<span class="glyphicon glyphicon-download" id="buyItemButton"></span>Own Item: ' + (parseFloat(cost / 100).toFixed(2)) + ' HFC');
+                $('#side-info').find('.btn').first().html('<span class="glyphicon glyphicon-download" id="buyItemButton"></span>Own Item: ' + cost + ' HFC');
+
```
Comment (zfox23) [inline scripts/system/html/js/marketplacesInject.js]: Extra whitespace here.

I'd recommend reverting this file - there are no changes to it besides whitespace changes, and it just clutters up the PR. What do you think?

Besides this, this PR LGTM.

**Comment by zfox23** [comment]
build this please

**Comment by zfox23** [comment]
GUSTAVOOOOOO!!!!!

File: scripts/system/html/js/marketplacesInject.js (hunk 2 above)
Comment (davidkelly) [reply scripts/system/html/js/marketplacesInject.js]: the cost is different

Comment (zfox23) [reply scripts/system/html/js/marketplacesInject.js]: :joy: DUH! Thanks for pointing that out.

Well, there _is_ some extra whitespace there... ;)

**Comment by zfox23** [comment]
[Always just show HFC "pennies".](https://trello.com/c/Zf2UGa46/357-always-just-show-hfc-pennies)

**Comment by zfox23** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3001/HighFidelity-Beta-PR11289-cdadaa8c9c220b15d868095d8e68b43432a2c0fa.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3004/HighFidelity-Beta-PR11289-1eb503f526b548705c3695cdfd4932562726cc52.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3020/HighFidelity-Beta-PR11289-655dc61ff270f717755896b33b92674023e04fdf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3020/HighFidelity-Beta-PR11289-655dc61ff270f717755896b33b92674023e04fdf.exe

**Pull request merged**